JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -202,8 +159,12 @@ private fun InterestsScreenContent(
                 elevation = 0.dp
             )
         }
-    ) {
-        TabContent(tab, onTabChange, tabContent)
+    ) { innerPadding ->
+        val screenModifier = Modifier
+            .padding(innerPadding)
+            .navigationBarsPadding(start = false, bottom = false)
```

Comment:
We sometimes do want to have the start navigation bars padding applied here, if the nav rail is hidden. Otherwise the content is obscured behind the navigation bar:

Reply:
We also want to apply the bottom padding from navigation bars, otherwise the bottom of the content can be obscured:

![device-2021-09-17-140741](https://user-images.githubusercontent.com/4217560/133853746-7a92856f-2eeb-4b1a-a213-64bab8f9adc5.png)

Ideally this would be applied as content padding, so that we take advantage of a translucent navigation bar.

Reply:
Fixed, thanks!